tests/cpp/tts/test_magpietts_cached_attention.cpp

```
@@ -0,0 +1,210 @@
+                output[qkv_index(d, head, batch)] = context / denominator;
+            }
+        }
+    }
+    return output;
+}
+
+}  // namespace
+
+int
+main() {
+    ggml_backend_t backend = ggml_backend_cuda_init(0);
```

Clang (14.0.6)

[warning] variable 'backend' is not initialized

(cppcoreguidelines-init-variables)